

============================== 02:38:37.926912 | 375f52ee-7b9e-47ca-b9e5-cf0c02336893 ==============================
[0m02:38:37.926912 [info ] [MainThread]: Running with dbt=1.6.6
[0m02:38:37.933494 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/ez/Library/CloudStorage/GoogleDrive-ezchamba@gmail.com/My Drive/EzTech/Projects/Learning/dbt-project', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'warn_error': 'None', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt run', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'False'}
[0m02:38:37.934711 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/ez/Library/CloudStorage/GoogleDrive-ezchamba@gmail.com/My Drive/EzTech/Projects/Learning/dbt-project/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m02:38:37.937025 [debug] [MainThread]: Command `dbt run` failed at 02:38:37.936703 after 0.09 seconds
[0m02:38:37.937581 [debug] [MainThread]: Flushing usage events
